ext/php5/ddtrace.c

```diff
@@ -1762,6 +1759,40 @@ static PHP_FUNCTION(dd_trace_compile_time_microseconds) {
     RETURN_LONG(ddtrace_compile_time_get(TSRMLS_C));
 }
 
+static PHP_FUNCTION(set_priority_sampling) {
+    UNUSED(return_value_used, this_ptr, return_value_ptr);
+    bool global = false;
+    long priority;
+
+    if (zend_parse_parameters_ex(ZEND_PARSE_PARAMS_QUIET, ZEND_NUM_ARGS() TSRMLS_CC, "l|b", &priority, &global) ==
+        FAILURE) {
+        ddtrace_log_debug("Expected an integer and an optional boolen");
+        RETURN_FALSE;
+    }
+
+    if (global || !DDTRACE_G(root_span)) {
+        DDTRACE_G(default_priority_sampling) = priority;
+    } else {
+        ddtrace_set_prioritySampling_on_root(priority TSRMLS_CC);
+    }
+}
+
+static PHP_FUNCTION(get_priority_sampling) {
+    UNUSED(return_value_used, this_ptr, return_value_ptr);
+    zend_bool global = false;
+
+    if (zend_parse_parameters_ex(ZEND_PARSE_PARAMS_QUIET, ZEND_NUM_ARGS() TSRMLS_CC, "|b", &global) == FAILURE) {
+        ddtrace_log_debug("Expected an optional boolen");
+        RETURN_FALSE;
```

Comment:
The prototype for the callers is `int|null`, which doubly makes me think we should return null instead of false here. I have another review comment about it.